cmd/zed/zed_conf.c

```diff
@@ -367,7 +367,7 @@ zed_conf_scan_dir(struct zed_conf *zcp)
 		n = snprintf(pathname, sizeof (pathname),
 		    "%s/%s", zcp->zedlet_dir, direntp->d_name);
 		if ((n < 0) || (n >= sizeof (pathname))) {
-			zed_log_msg(LOG_WARNING, "Failed to stat \"%s\": %s",
+			zed_log_msg(LOG_WARNING, "Failed to snprintf \"%s\": %s",
```

Comment:
While it's true this "new" warning is more exact it also doesn't explain the context of the failing `snprintf()`, which i think is less useful for someone reading the log message; on the other hand the "old" warning is much more explicit about our intentions (we are going to `stat()` a path) and the reason why this failed (`ENAMETOOLONG`).

Reply:
I agree, the intent here is the important thing to report not the specific library call which failed.  Let's leave this code as-is.

Reply:
understood. thanks